docs/celsius-converter-exception/dune

```diff
@@ -0,0 +1,6 @@
+(melange.emit
```

Comment:
An alternative "design" for this could be to have just `library` stanzas on each chapter folder, with the `(modes melange)`, and then in `docs/dune` have a single `melange.emit` that includes all the libraries.

The main upside is that things like `module_systems` or `target` are only defined once.

Reply:
I think I'll keep the current layout for now. Some chapters will explicitly be about creating libraries and not apps, so I want to keep the distinction clear.